pandas/tests/extension/base/groupby.py

```diff
@@ -56,6 +56,40 @@ def test_groupby_agg_extension(self, data_for_grouping):
         result = df.groupby("A").first()
         self.assert_frame_equal(result, expected)
 
+    def test_groupby_agg_extension_timedelta_cumsum_with_named_aggregation(self):
+        # GH#41720
+        expected = pd.DataFrame(
+            {
+                "td": {
+                    0: pd.Timedelta("0 days 01:00:00"),
+                    1: pd.Timedelta("0 days 01:15:00"),
+                    2: pd.Timedelta("0 days 01:15:00"),
+                }
+            }
+        )
+        df = pd.DataFrame(
+            {
+                "value": pd.Series(["1", "2", "3"], dtype="int64"),
+                "ts": pd.Series(
+                    [
+                        "2021-01-01 00:30:00",
+                        "2021-01-01 00:45:00",
+                        "2021-01-01 02:00:00",
+                    ],
+                    dtype="datetime64[ns]",
+                ),
+                "td": pd.Series(
+                    ["0 days 01:00:00", "0 days 00:15:00", "0 days 01:15:00"],
+                    dtype="timedelta64[ns]",
+                ),
+                "amount": pd.Series(["10", "20", "30"], dtype="int64"),
+                "grps": pd.Series(["a", "a", "b"], dtype="object"),
```

Comment:
Can just specify as a list here: `"grps": ["a", "a", "b"],`

Reply:
Ohh alright, I wasn't entirely sure, but checking it, it seems to work, so I updated it here: 6d8ae022355020b73c98e22c216381b81569f0b6